ops/services/app_gateway/main.tf

```diff
@@ -0,0 +1,138 @@
+locals {
+  static_backend_pool         = "${var.name}-${var.env}-be-static"
+  static_backend_http_setting = "${var.name}-${var.env}-be-static"
+}
+
+resource "azurerm_public_ip" "static_gateway" {
+  name                = "${var.name}-${var.env}-gateway"
+  resource_group_name = var.resource_group_name
+  location            = var.resource_group_location
+  allocation_method   = "Static"
+  sku                 = "Standard"
+
+  tags = var.tags
+}
+
+resource "azurerm_application_gateway" "load_balancer" {
+  name                = "${var.name}-app-gateway"
+  resource_group_name = var.resource_group_name
+  location            = var.resource_group_location
+
+  sku {
+    name = var.sku_name
+    tier = var.sku_tier
+  }
+
+  gateway_ip_configuration {
+    name      = "${var.name}-${var.env}-gateway-ip-config"
+    subnet_id = var.subnet_id
+  }
+
+  frontend_port {
+    name = "${var.name}-fe-port"
+    port = 80
+  }
+
+  frontend_ip_configuration {
+    name                 = "${var.name}-fe-ip-config"
+    public_ip_address_id = azurerm_public_ip.static_gateway.id
+  }
+
+  # ------- Static -------------------------
+  backend_address_pool {
+    name  = local.static_backend_pool
+    fqdns = ["simple-report-dev.azureedge.net"]
+  }
+
+  backend_http_settings {
+    name                                = local.static_backend_http_setting
+    cookie_based_affinity               = "Disabled"
+    port                                = 80
+    protocol                            = "Http"
+    request_timeout                     = 20
+    pick_host_name_from_backend_address = true
+    probe_name                          = "static"
+  }
+
+  http_listener {
+    name                           = "${var.name}-static"
+    frontend_ip_configuration_name = "${var.name}-fe-ip-config"
+    frontend_port_name             = "${var.name}-fe-port"
+    //    host_name                      = "dev.simplereport.org"
+    protocol = "Http"
+  }
+
+  request_routing_rule {
+    name                       = "${var.name}-routing-static"
+    rule_type                  = "PathBasedRouting"
+    http_listener_name         = "${var.name}-static"
+    backend_address_pool_name  = local.static_backend_pool
+    backend_http_settings_name = local.static_backend_http_setting
+    url_path_map_name          = "${var.env}-urlmap"
+  }
+
+  probe {
+    name                                      = "static"
+    interval                                  = 10
+    path                                      = "/"
```

Comment:
This should probably point to /actuator/health

Reply:
ah yes, this is for the static app specifically not API, but good point on updating the api health check to that route